api/src/team/scripts/send-organization-invitations.script.js

```diff
@@ -54,24 +56,47 @@ export class SendOrganizationInvitationsScript extends Script {
         },
       },
     });
+    this.failedLines = 0;
+    this.successLines = 0;
+    this.actualLine = 0;
   }
 
-  async handle({ options, logger, sendOrganizationInvitation = usecases.createOrganizationInvitationByAdmin }) {
+  async handle({
+    options,
+    logger,
+    sendOrganizationInvitation = usecases.createOrganizationInvitationByAdmin,
+    checkMail = checkDomainIsValid,
```

Comment:
Pourquoi créer une nouvelle propriété `checkMail` et ne pas avoir tout simplement la propriété nommée `checkDomainIsValid` ?

```suggestion
    checkDomainIsValid,
```